userenvs/opensuse.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
You weren't supposed to commit this one...

Reply:
I thought I also did a 'remove' for the commit to netgate this out, but maybe not.